tests/test1.in

```diff
@@ -33,12 +33,12 @@ end
 # Simple alias tests
 
 function foo
-    echo >foo.txt $argv
+    echo >/tmp/fish_foo.txt $argv
```

Comment:
Okay, I'll create *test/{data,home,root,temp}* directories. I should have that change available tomorrow (time to go to bed).